src/cirq_iqm/iqm_remote.py

```diff
@@ -66,12 +66,12 @@ def run_sweep(
             params: cirq.Sweepable,
             repetitions: int = 1,
     ) -> list[cirq.Result]:
-        """Samples from the given quantum circuit.
+        """Samples from, i.e. executes, the given quantum circuit.
```

Comment:
"Samples from the given quantum circuit." comes directly from the the parent class, not sure if changing it brings more clarity.

Reply:
Let's keep it as it was. The user won't call this method directly anyway.

Reply:
AFAIK this is supposed to be one of the functions the user may call directly. But anyway, no point in copying the superclass docstring verbatim, I'll just delete it here. IDEs and our docs generation are able to pick up the docstring from the superclass.

Reply:
Docstring removed here.

Reply:
Turns out the docstring was not verbatim copy of the superclass one.

Reply:
Reworded.